core/modules/views/tests/views_upgrade.test

```diff
@@ -15,11 +15,6 @@ require_once BACKDROP_ROOT . '/core/modules/views/tests/views_query.test';
 class ViewsUpgradeTestCase extends ViewsSqlTest {
   protected function setUp(array $modules = array()) {
     parent::setUp($modules);
-
-    // To import a view the user needs use PHP for settings rights, so enable php module.
-    // @TODO The php module does not exist in core in Backdrop!
-    module_enable(array('php'));
-    $this->resetAll();
   }
```

Comment:
My OOP isn't very strong so forgive me if I'm mistaken, but if we remove the custom bit then do we even need to keep this `setUp()` function if all it does is call the parent `setUp()` function? If we remove this function altogether, won't the parent function still be called? Or is that not the case...?

Reply:
You are right. This override is not needed here. I'll remove it.